app/logic/emailHandler.py

```diff
@@ -0,0 +1,84 @@
+from flask import Flask, request, render_template
+import yaml, os
+from flask_mail import Mail, Message
+from app.models.interest import Interest
+from app.models.user import User
+from app.models.eventParticipant import EventParticipant
+from app.models.emailTemplate import EmailTemplate
+from app import app
+import sys
+import webbrowser
+from pathlib import Path
+
+def getInterestedEmails(programID = None):
+    """ Gets emails of students interested in a program. """
+    volunteersToEmail = User.select().join(Interest).where(Interest.program == programID)
+    return [user.email for user in volunteersToEmail]
+
+def getParticipantEmails(eventID = None):
+    """ Gets emails of students participating in an event. """
+    volunteersToEmail = User.select().join(EventParticipant).where(EventParticipant.event == eventID)
+    return [user.email for user in volunteersToEmail]
+
+class emailHandler():
+    """ A class for email setup and configuring the correct data to send. """
+    def __init__(self, mail, emailInfo):
+        app.config.update(
+            MAIL_SERVER=app.config['mail']['server'],
+            MAIL_PORT=app.config['mail']['port'],
+            MAIL_USERNAME= app.config['mail']['admin_username'],
+            MAIL_PASSWORD= app.config['mail']['admin_password'],
+            REPLY_TO_ADDRESS= app.config['mail']['reply_to_address'],
+            MAIL_USE_TLS=app.config['mail']['tls'],
+            MAIL_USE_SSL=app.config['mail']['ssl'],
+            MAIL_DEFAULT_SENDER=app.config['mail']['default_sender'],
+            #ALWAYS_SEND_MAIL=default['ALWAYS_SEND_MAIL']
+            MAIL_OVERRIDE_ALL=app.config['mail']['override_addr']
+        )

```

Comment:
there's no reason to update all of these config values. You aren't even using them anymore.

Reply:
only updates the necessary config values now.